### ISSUE TITLE
Error when use the subscription confirm

### ISSUE THREAD

**Issue by astehlik** [astehlik/typo3-extension-formhandler_subscription#11]
---

Author Name: **Kay Röseler** (Kay Röseler)
Original Redmine Issue: 44385, https://forge.typo3.org/issues/44385
Original Date: 2013/01/08
Original Assignee: Alexander Stehlik

---

Hello,

if I use the current TER version in TYPO3 6.0, I get an error when I try to confirm the registration.

To correct this error, the file Tx_FormhandlerSubscription_Utils_AuthCode.php in the line 108 of the code can be changed.

instead of

```
$this->TSFE = $GLOBALS['TSFE'];

```

it should read

```
$this->tsfeUser = $GLOBALS['TSFE']->fe_user;

```

Greetings
Kay

**Comment by astehlik** [comment]
---

Original Redmine Comment
Author Name: **Klaus Weidenbach** (Klaus Weidenbach)
Original Date: 2013/01/11 14:57:51 +0100

---

I can confirm that this patch fixes also problems in TYPO3 CMS 4.6 when trying to access the subscription update form.

**Comment by astehlik** [comment]
---

Original Redmine Comment
Author Name: **Alexander Stehlik** (Alexander Stehlik)
Original Date: 2013/01/11 17:30:43 +0100

---

Hi Guys,

thanks for the report.

An updated version was released to the TER and is now available at "GitHub":https://github.com/astehlik/typo3-extension-formhandler_subscription

Cheers,
Alex
